pkg/cmd/get/get_workflow.go

```diff
@@ -1,136 +0,0 @@
-package get
-
-import (
-	"strings"
-
-	"github.com/jenkins-x/jx/pkg/cmd/helper"
-
-	"github.com/jenkins-x/jx/pkg/client/clientset/versioned"
-	"github.com/jenkins-x/jx/pkg/cmd/opts"
-	"github.com/jenkins-x/jx/pkg/cmd/templates"
-	"github.com/jenkins-x/jx/pkg/log"
-	"github.com/jenkins-x/jx/pkg/workflow"
-	"github.com/spf13/cobra"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-)
-
-// GetWorkflowOptions containers the CLI options
-type GetWorkflowOptions struct {
-	GetOptions
-
-	Name string
-}
-
-var (
-	getWorkflowLong = templates.LongDesc(`
-		Display either all the workflows or a specific workflow
-`)
-
-	getWorkflowExample = templates.Examples(`
-		# List all the available workflows
-		jx get workflow
-
-		# Display a specific workflow
-		jx get workflow -n default
-	`)
-)
-
-// NewCmdGetWorkflow creates the new command for: jx get env
-func NewCmdGetWorkflow(commonOpts *opts.CommonOptions) *cobra.Command {
-	options := &GetWorkflowOptions{
-		GetOptions: GetOptions{
-			CommonOptions: commonOpts,
-		},
-	}
-	cmd := &cobra.Command{
-		Use:     "workflows",
-		Short:   "Display either all the available workflows or a specific workflow",
-		Aliases: []string{"workflow"},
-		Long:    getWorkflowLong,
-		Example: getWorkflowExample,
-		Run: func(cmd *cobra.Command, args []string) {
-			options.Cmd = cmd
-			options.Args = args
-			err := options.Run()
-			helper.CheckErr(err)
-		},
-	}
-	cmd.Flags().StringVarP(&options.Name, "name", "n", "", "The name of the workflow to display")
-
-	options.AddGetFlags(cmd)
-	return cmd
-}
-
-// Run implements this command
-func (o *GetWorkflowOptions) Run() error {
-	jxClient, ns, err := o.JXClientAndAdminNamespace()
-	if err != nil {
-		return err
-	}
-
-	name := o.Name
-	if name == "" {
-		if len(o.Args) > 0 {
-			name = o.Args[0]
-		}
-	}
-
-	if name != "" {
-		return o.getWorkflow(name, jxClient, ns)
-	}
-
-	workflows, err := jxClient.JenkinsV1().Workflows(ns).List(metav1.ListOptions{})
```

Comment:
I am not sure about this _workflow_ thing. If we remove the get command don't we have to delete everything around it as well, aka the WorkFlow type. The docs says `WorkflowSpec is the specification of the pipeline activity`. 

I guess this ties also into the workflow controller. I am still not sure what that's meant to do and whether this is still used atm in any form.

@abayer ^^

Reply:
I literally have no clue with the workflow controller is for - it's not used in Tekton-based installs, though, and I don't see it in BDD tests of static masters, so we probably can get rid of it? @jstrachan or @rawlingsj would know more about what it was intended to be doing than I do.

Reply:
No idea what this `workflow` thing is, I think @rawlingsj tried to remove it from the Jenkins X platform because he thought it's also unused.

Reply:
I removed it from this PR for now, we can come back and review this

Reply:
Spoke to @jstrachan, it's an experiment we introduced in the early days to avoid lots of huge jenkins build pods being around waiting for the promotion to happen. The relevant PR is https://github.com/jenkins-x/jx/pull/1476